### PR TITLE
Add a redundant else block checker

### LINKED ISSUE #7: new checker: unneededElseBlock
### Expected behavior

The `lintElses` checker examines `else` blocks and complains about any `else` block whose `if` block ends with a terminating (`return`/`os.Exit`) statement.

### PR DESCRIPTION
**Pull request by epgoranova**
Closes #7

### CODE REVIEW

File: checker/redundant_else.go [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+	report *Report) {
+
+	stmt := node.(*ast.IfStmt)
+
+	if stmt.Else == nil {
+		return
+	}
+
+	var termStmt string
+
```
Comment (s2gatev) [inline checker/redundant_else.go]: Remove empty line.

File: checker/redundant_else.go [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+}
+
+// Check implements the NodeChecker interface.
+func (c *RedundantElseChecker) Check(
+	node ast.Node,
+	content string,
+	report *Report) {
+
+	stmt := node.(*ast.IfStmt)
+
```
Comment (s2gatev) [inline checker/redundant_else.go]: Remove empty line.

File: checker/redundant_else_test.go [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+
+import (
+	"testing"
+
+	. "github.com/s2gatev/lingo/checker"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestRedundantElseChecker(t *testing.T) {
```
Comment (s2gatev) [inline checker/redundant_else_test.go]: panic?

File: checker/redundant_else_test.go [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+	type test struct {
```
[the remark this answers is not in the document]
Comment (catiepg) [reply checker/redundant_else_test.go]: Done.

File: checker/redundant_else.go (hunk 1 above)
Comment (catiepg) [reply checker/redundant_else.go]: Removed.

File: checker/redundant_else.go (hunk 2 above)
Comment (catiepg) [reply checker/redundant_else.go]: Removed.

File: checker/redundant_else_test.go (hunk 3 above)
Comment (catiepg) [reply checker/redundant_else_test.go]: Added.

**Review by sgatev — APPROVED** [review]

**Pull request merged**